### ISSUE TITLE
Failed to convert to Tensorflow Lite - TensorFlowMax, TensorFlowMinimum, TensorFlowSum not supported

### ISSUE THREAD

**Issue by lun0522** [tensorflow/tensorflow#20042]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: MacOS 10.13.4
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.8.0
- **Python version**: 3.6.5
- **Bazel version (if compiling from source)**: 0.14.0-homebrew
- **GCC/Compiler version (if compiling from source)**: Apple LLVM version 9.1.0 (clang-902.0.39.2)
- **CUDA/cuDNN version**: n/a
- **GPU model and memory**: n/a
- **Exact command to reproduce**: bazel-bin/tensorflow/contrib/lite/toco/toco --input_file=.tfmodel --output_file=/Users/plun/Downloads/mobilenet_v2.tflite --input_shape=1,224,224,3 --input_array=input_1 --output_array=reshape_2/Reshape --inference_type=FLOAT

I am trying to convert a MobileNetV2 model to Tensorflow Lite. I was able to convert the model here: https://github.com/tensorflow/models/tree/master/research/slim#pre-trained-models, but when I try to convert a model trained by myself using Keras, the following error occurs:

"Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If you have a custom implementation for them you can disable this error with --allow_custom_ops. Here is a list of operators for which you will need custom implementations: **TensorFlowMax**, **TensorFlowMinimum**, **TensorFlowSum**."

I ran `convert_variables_to_constants` to freeze the model, and ` bazel-bin/tensorflow/contrib/lite/toco/toco` to do conversion. I inspected the structure of the frozen graph, and found that TensorFlowMax, TensorFlowMinimum, TensorFlowSum are in relu layers and softmax layer. Are they really unsupported yet, or did I miss some steps?

Complete console output:
![screen shot 2018-06-14 at 4 39 54 pm](https://user-images.githubusercontent.com/24245729/41443425-9e22e676-6ff1-11e8-90eb-583c9e209698.png)

**Comment by lun0522** [comment]
I found out how to avoid using these layers.

**Comment by lun0522** [comment]
I found out how to avoid using these layers. 

Keras may use **TensorFlowMinimum** for relu6. In its source code:
<img width="754" alt="screen shot 2018-06-15 at 9 47 44 am" src="https://user-images.githubusercontent.com/24245729/41481524-297a2e1a-7087-11e8-8521-eda7b4135b19.png">
Keras implements it as a combination of a normal relu and a minimum layer. To avoid this, use tf.nn.relu6 instead.

Keras may use **TensorFlowMax** and **TensorFlowSum** for softmax. In its source code:
<img width="870" alt="screen shot 2018-06-15 at 9 52 21 am" src="https://user-images.githubusercontent.com/24245729/41481604-7f9149e6-7087-11e8-8a71-4bfd0c23abc0.png">
I forgot to reshape the input tensor, and the dimension of it was greater than 2, so Keras breaks it down to several sublayers. We can avoid this by reshaping the tensor before softmax.

**Comment by seovchinnikov** [comment]
got the same, in my implementation it was clipvalue operation, Ive converted it to min(relu,6), but it has to be relu6 op instead
